src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/WinResources.properties

```diff
@@ -34,6 +34,7 @@ resource.executable-properties-template=Template for creating executable propert
 resource.setup-icon=setup dialog icon
 resource.post-app-image-script=script to run after application image is populated
 resource.post-msi-script=script to run after msi file for exe installer is created
+resource.wxl-file=WiX localization file
```

Comment:
This needs to be added to all WinResources*.properties files.

Reply:
Good point. We have a discussion of how l10n files should be changed at https://github.com/openjdk/jdk/pull/9753. Can you confirm that if some property is missing in a resource bundle its value is picked from the English resource bundle?

Reply:
I've just pushed the commit with EN entries for all files before reading the latest comment. I'll verify the behaviour with `.wxl` and `.properties` files and will comment in this issue.

Reply:
I believe for the `.properties` case, modifications for the localized files are not required, as ResourceBundles fall back to the root (English) bundle.